extensions/wikia/WikiFactory/templates/variable.tmpl.php

```diff
@@ -99,19 +93,15 @@
 
 <?php elseif( $variable->cv_variable_type == "integer"): ?>
 
-	<input type="text" name="varValue" id="varValue" value="<?php echo unserialize( $variable->cv_value ) ?>" size="40" maxlength="255" />
+	<input type="text" name="varValue" id="varValue" value="<?php echo WikiFactory::renderValue( $variable ) ?>" size="40" maxlength="255" />
```

Comment:
it doesn't seem like a regression, but HTML-escaping the variables is missing is the template